observer/lib/observer.js

```diff
@@ -96,3 +96,28 @@ export const observeScheduledRewards = async (pgPools, ieContract, fetch = globa
 function isEventLog (logOrEventLog) {
   return 'args' in logOrEventLog
 }
+
+export const observeRetrievalResultStatus = async (pgPoolStats, influxQueryApi) => {
+  const rows = await influxQueryApi.collectRows(`
+    import "strings"
+    from(bucket: "spark-evaluate")
+      |> range(start: 0)
```

Comment:
Keep in mind we have a 90-day retention for the spark-evaluate InfluxDB bucket. Is your solution going to handle this?

It also seems to be a bit inefficient to always query all 90 days when in most cases, we need only data for yesterday.

Having said that, I guess we don't worry about the performance (and the InfluxDB bill) right now.

Reply:
> Keep in mind we have a 90-day retention for the spark-evaluate InfluxDB bucket. Is your solution going to handle this?

This solution works precisely because we have the 90 day retention. Without it, the query would be unbounded.

> It also seems to be a bit inefficient to always query all 90 days when in most cases, we need only data for yesterday.
> Having said that, I guess we don't worry about the performance (and the InfluxDB bill) right now.

I agree, I'm going for the pragmatic solution here.

Reply:
> > Keep in mind we have a 90-day retention for the spark-evaluate InfluxDB bucket. Is your solution going to handle this?
> 
> This solution works precisely because we have the 90 day retention. Without it, the query would be unbounded.

Would you mind adding a code comment to explain this to future readers?

Reply:
c62756a